HW_6/Makarov_V/HW6_3.py

```diff
@@ -0,0 +1,13 @@
+def word_count(text):
+    """
+    This function counts amount
+    of each given symbol
+    in the text
+    """
+    count = {}
```

Comment:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result